### PR TITLE
CAL-420 updated docs appendix for catalog taxonomy

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

-reorganized appendix for metacard attributes
-separated catalog taxonomy attribute tables(taxonomy only) from mappings between catalog and standards.
- Added links from attributes in mappings to attribute definition tables, where they exist.

#### Who is reviewing it? 

@emmberk @brendan-hofmann 
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)

@beyelerb
@brendan-hofmann
@clockard
@lessarderic
@ricklarsen - Documentation
@shaundmorris

#### How should this be tested?

-build successfully
- links work between attributes

#### What are the relevant tickets?
[DDF-3633](https://codice.atlassian.net/browse/DDF-3633)
[CAL-420](https://codice.atlassian.net/browse/CAL-420)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_metadataAttributes/nsili-attributes-table.adoc [hunk 1]
```diff
@@ -34,14 +34,14 @@
...
-|`modified`
+|<<modified,modified>>
 |none
 |Date
 
 |NSIL_CARD
 |identifier
-|add a value to `metacard.associations.related` if the identifier refers to an associated metacard (i.e. the card is an association node); +
-else, `id`
+|add a value to <<metacard.associations.related,metacard.associations.related>> if the identifier refers to an associated metacard (i.e. the card is an association node); +
```
Comment (emmberk) [inline distribution/docs/src/main/resources/content/_metadataAttributes/nsili-attributes-table.adoc]: This link doesn't work for me.

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_metadataAttributes/nsili-attributes-table.adoc]: testing a fix now

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_metadataAttributes/nsili-attributes-table.adoc]: fixed

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
Docs-only change.

**Pull request merged**